README.md

```diff
@@ -1,12 +1,32 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Individual project, creating a Business Site including a hero-image or -video, a form with some elements and an accordion. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I planned to have a video as "hero-image" so I started to find a suitable video for the format, but it was hard to find. I changed my mind and choosed three different nice pink pictures to use as hero-image. I designed the whole page based on the colors form the hero-image. So the three different device formats have differents colors on the rest och the page, different box-shadows and different border-radius. 
```

Comment:
It's great to see that you've really thought about this, made a plan and executed it very well, BRAVO! 🌟